Jenkinsfile

```diff
@@ -106,10 +106,18 @@ podTemplate(
             println("This version: " + currentVersion)
             def versionExists = sh(returnStdout: true, script: 'pipenv run python3 cognite/client/utils/_version_checker.py -p cognite-sdk -v ' + currentVersion).trim()
             println("Version Exists: " + versionExists)
+
             if (env.BRANCH_NAME == 'master' && versionExists == 'no') {
                 stage('Release') {
                     sh("pipenv run twine upload --config-file /pypi/.pypirc dist/*")
                 }
+                stage('After release') {
```

Comment:
This is the name that shows up in jenkins right, can we name it something sensible like 'Update code snippets'

Reply:
done